### PR TITLE
Functional testing documentation

### PR DESCRIPTION
**Pull request by jakub-dzon**
```release-note
NONE
```
Signed-off-by: Jakub Dzon <jdzon@redhat.com>

### CODE REVIEW

**Review by machacekondra — APPROVED** [review]

**Review by pkliczewski — COMMENTED** [review]
It looks great, can you add a reference to this file from top level readme?

**Review by pkliczewski — APPROVED** [review]

File: docs/functional-tests.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Functional tests for the vm-import-operator are listed below.
+
+## Basic VM
+| Test description | Implemented |
+| :---------------- | :-----------: |
+| Basic VM import without resource mapping should create stopped VM | &check; |
+| Basic import without resource mapping should create started VM | &check; |
+| Basic import with in-CR resource mapping should create running VM for storage domain mapping | &check; |
+| Basic import with in-CR resource mapping should create running VM for storage disk mapping | &check; |
+| Basic import with in-CR resource mapping should create running VM for storage disk mapping | &check; |
```
Comment (masayag) [inline docs/functional-tests.md]: this one is exactly as the one above it

File: docs/functional-tests.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Functional tests
+
+Functional tests for the vm-import-operator are listed below.
```
Comment (masayag) [inline docs/functional-tests.md]: Could you add link to the tests directory ?

File: docs/functional-tests.md (hunk 1 above)
Comment (jakub-dzon) [reply docs/functional-tests.md]: Done.

File: docs/functional-tests.md (hunk 2 above)
Comment (jakub-dzon) [reply docs/functional-tests.md]: Done

**Review by masayag — APPROVED** [review]

**Pull request merged**